lib/web_ui/lib/src/engine/text_editing/text_editing.dart

```diff
@@ -101,31 +101,30 @@ class EngineAutofillForm {
     // If there is only one text field in the autofill model, `fields` will be
     // null. `focusedElementAutofill` contains the information about the one
     // text field.
-    final bool singleElement = (fields == null);
-    final AutofillInfo focusedElement =
-        AutofillInfo.fromFrameworkMessage(focusedElementAutofill);
-    final Map<String, html.HtmlElement> elements = <String, html.HtmlElement>{};
-    final Map<String, AutofillInfo> items = <String, AutofillInfo>{};
+    final Map<String?, html.HtmlElement> elements = <String?, html.HtmlElement>{};
```

Comment:
I might be missing something, I don't understand the usage of String? as a map key. 

Does that mean the key is nullable?

Reply:
Done. Most of the conversion is syntactic. Prior to null safety everything was nullable. The goal of this PR is to convert to the new syntax (i.e. add `?` to what's nullable). Converting to non-null is nice-to-have, but it's not realistic to clean up everything in one PR. I expect most of the actual refactoring will happen in many small PRs following this one.